editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -72,13 +112,47 @@ impl ToolMetadata for PathTool {
 
 impl LayoutHolder for PathTool {
 	fn layout(&self) -> Layout {
-		Layout::WidgetLayout(WidgetLayout::default())
+		if let Some(SingleSelectedPoint { coordinates: DVec2 { x, y }, .. }) = self.tool_data.single_selected_point {
+			let x_loc = NumberInput::new(Some(x))
+				.unit(" px")
+				.label("x")
```

Comment:
Uppercase please (and `"y"`) also. Also, if you can set `.min_width(120)` that would help avoid the numbers from getting cut off.

Reply:
just saw this, fixing and pushing. sorry for the rush.